conversation_reconstruction_dataflow/ingest_revisions/dataflow_main.py

```diff
@@ -131,5 +110,27 @@ def process(self, element):
 
 if __name__ == '__main__':
   logging.getLogger().setLevel(logging.INFO)
-  run()
+  parser = argparse.ArgumentParser()
```

Comment:
Why did you move the argument parsing to main?

Reply:
I was trying to find the reason why it was not parsing one of the arguments, this is not the reason in the end. Should I move it back?